### PR TITLE
Remove unnecessary nuget.config file

### PR DESCRIPTION
**Pull request by weshaggard**

### CODE REVIEW

**Review by jsquire — APPROVED** [review]

**Review by petrsvihlik — APPROVED** [review]

**Review by JoshLove-msft — APPROVED** [review]

**Comment by weshaggard** [comment]
The codecheck issue looks unrelated so going ahead and merging this to unblock all the other pipelines. I will double check the core pipeline after merging.

**Pull request merged**
